### ISSUE TITLE
Gives an error in log if there is no config file

### ISSUE THREAD

**Issue by FewerFlaws** [Bstn1802/AutoReconnect#8]
Pretty small error since it still creates the default config file, but worth correcting.  Could be changed to a warning if the behaviour is desireable.

Here's the error:
`[22:53:18] [main/ERROR]: [AutoReconnect] NoSuchFileException: E:\minecraft\multiMC\mmc-stable-win32\MultiMC\instances\Single Mod Test\.minecraft\config\autoreconnect.json`

**Comment by FewerFlaws** [comment]
Full log just in case it helps:
```
MultiMC version: 0.6.11-1430


Minecraft folder is:
E:/minecraft/multiMC/mmc-stable-win32/MultiMC/instances/Single Mod Test/.minecraft


Java path is:
C:/Program Files (x86)/Common Files/Oracle/Java/javapath/javaw.exe


Java is version 1.8.0_271, using 64-bit architecture.


Main Class:
  net.fabricmc.loader.launch.knot.KnotClient

Native path:
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/instances/Single Mod Test/natives

Traits:
traits FirstThreadOnMacOS
traits XR:Initial

Libraries:
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.2.2/lwjgl-glfw-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.2.2/lwjgl-jemalloc-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.2.2/lwjgl-openal-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.2.2/lwjgl-opengl-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.2.2/lwjgl-stb-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.2.2/lwjgl-tinyfd-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl/3.2.2/lwjgl-3.2.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/patchy/1.1/patchy-1.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/oshi-project/oshi-core/1.1/oshi-core-1.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/java/dev/jna/jna/4.4.0/jna-4.4.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/java/dev/jna/platform/3.4.0/platform-3.4.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/ibm/icu/icu4j/66.1/icu4j-66.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/javabridge/1.0.22/javabridge-1.0.22.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/sf/jopt-simple/jopt-simple/5.0.3/jopt-simple-5.0.3.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/io/netty/netty-all/4.1.25.Final/netty-all-4.1.25.Final.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/google/guava/guava/21.0/guava-21.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/commons/commons-lang3/3.5/commons-lang3-3.5.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/commons-io/commons-io/2.5/commons-io-2.5.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/commons-codec/commons-codec/1.10/commons-codec-1.10.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/brigadier/1.0.17/brigadier-1.0.17.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/datafixerupper/4.0.26/datafixerupper-4.0.26.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/google/code/gson/gson/2.8.0/gson-2.8.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/authlib/2.1.28/authlib-2.1.28.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/httpcomponents/httpclient/4.3.3/httpclient-4.3.3.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/httpcomponents/httpcore/4.3.2/httpcore-4.3.2.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/it/unimi/dsi/fastutil/8.2.1/fastutil-8.2.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/logging/log4j/log4j-api/2.8.1/log4j-api-2.8.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/apache/logging/log4j/log4j-core/2.8.1/log4j-core-2.8.1.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/text2speech/1.11.3/text2speech-1.11.3.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/intermediary/1.16.4/intermediary-1.16.4.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/tiny-mappings-parser/0.2.2.14/tiny-mappings-parser-0.2.2.14.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/sponge-mixin/0.8.2+build.24/sponge-mixin-0.8.2+build.24.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/tiny-remapper/0.3.0.70/tiny-remapper-0.3.0.70.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/access-widener/1.0.0/access-widener-1.0.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/fabric-loader-sat4j/2.3.5.4/fabric-loader-sat4j-2.3.5.4.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/google/jimfs/jimfs/1.2-fabric/jimfs-1.2-fabric.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/ow2/asm/asm/9.0/asm-9.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/ow2/asm/asm-analysis/9.0/asm-analysis-9.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/ow2/asm/asm-commons/9.0/asm-commons-9.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/ow2/asm/asm-tree/9.0/asm-tree-9.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/ow2/asm/asm-util/9.0/asm-util-9.0.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/fabric-loader/0.10.8/fabric-loader-0.10.8.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/minecraft/1.16.4/minecraft-1.16.4-client.jar

Native libraries:
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-glfw/3.2.2/lwjgl-glfw-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-jemalloc/3.2.2/lwjgl-jemalloc-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-openal/3.2.2/lwjgl-openal-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-opengl/3.2.2/lwjgl-opengl-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-stb/3.2.2/lwjgl-stb-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl-tinyfd/3.2.2/lwjgl-tinyfd-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/org/lwjgl/lwjgl/3.2.2/lwjgl-3.2.2-natives-windows.jar
  E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/com/mojang/text2speech/1.11.3/text2speech-1.11.3-natives-windows.jar

Mods:
  [✔️] advdebug-2.2.0
  [✔️] amecs-1.3.2+mc.1.16.3
  [✔️] antighost-1.16.4-fabric0.25.1-1.1.3
  [✔️] autoconfig1u-3.3.1
  [✔️] autoreconnect-1.1.0
  [✔️] blame-2.2.6
  [✔️] Cardinal-Components-API-2.7.10
  [✔️] cloth-api-1.4.9
  [✔️] config-2-4.8.3
  [✔️] dark-loading-screen-1.5.0
  [✔️] DisableCustomWorldsAdvice-1.2
  [✔️] dynamic-fps-1.2.1
  [✔️] fabric-api-0.29.3+1.16
  [✔️] fabric-chunkpregen-0.3.3
  [✔️] fabric-language-kotlin-1.4.21+build.1
  [✔️] fabriczero-0.2.0
  [✔️] fastbench-2.3
  [✔️] fastfurnace-2.3
  [✔️] fatxporbs-0.0.9+1.16.4
  [✔️] inventory-hotswap-1.16-1.1.1
  [✔️] item-model-fix-0.2.4
  [✔️] juke-fix-1.0.0
  [✔️] konkrete_fabric_1.1.0_MC_1.16.3-1.16.4
  [✔️] lithium-fabric-mc1.16.4-0.6.0
  [✔️] modmenu-1.14.13+build.19
  [✔️] mousewheelie-1.6.1+mc1.16.4
  [✔️] nbtcrafting-2.0.3+mc1.16.4
  [✔️] phosphor-fabric-mc1.16.3-0.7.0+build.10
  [✔️] randompatches-2.1.5-fabric
  [✔️] RoughlyEnoughItems-5.8.9
  [✔️] smoothboot-fabric-1.16.4-1.4.0
  [✔️] sodium-fabric-mc1.16.3-0.1.0
  [✔️] spark-fabric
  [✔️] YungsApi-1.16.4-Fabric-1

Params:
  --username  --version MultiMC5 --gameDir E:/minecraft/multiMC/mmc-stable-win32/MultiMC/instances/Single Mod Test/.minecraft --assetsDir E:/minecraft/multiMC/mmc-stable-win32/MultiMC/assets --assetIndex 1.16 --uuid  --accessToken  --userType  --versionType release

Window size: 854 x 480

Java Arguments:
[-XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump, -Xms4096m, -Xmx4096m, -Duser.language=en]


Minecraft process ID: 34156


Using onesix launcher.

[22:53:09] [main/INFO]: Loading for game Minecraft 1.16.4
[22:53:10] [main/WARN]: Warnings were found! 
 - Conflicting versions found for fabric-api-base: used 0.2.0+ab87788d3a, also found 0.2.0+9f0fb4b83a
 - Conflicting versions found for fabric-key-binding-api-v1: used 1.0.1+730711c63a, also found 1.0.1+730711c6a7
 - Mod 'AntiGhost' (antighost) recommends version 1.1.7 or later of mod modupdater, which is missing!
	 - You must install version 1.1.7 or later of modupdater.
 - Conflicting versions found for fabric-lifecycle-events-v1: used 1.2.0+ffb68a873a, also found 1.2.0+74cc3b20a7
 - Conflicting versions found for fabric-events-lifecycle-v0: used 0.2.0+16acbe5ba7, also found 0.2.0+6a2618f53a
[22:53:10] [main/INFO]: [FabricLoader] Loading 113 mods: fabric-keybindings-v0@0.2.0+6a2618f53a, roughlyenoughitems-api@5.8.9, cloth-common-events-v1@1.4.9, fabricloader@0.10.8, crowdin-translate@1.2, fabriczero@0.2.0, cloth-durability-bar-api-v1@1.4.9, fabric-dimensions-v1@2.0.1+9a6c75813a, cotton-client-commands@1.0.1+1.16-rc1, fabric-game-rule-api-v1@1.0.5+cbe9176f3a, fabric-api-base@0.2.0+ab87788d3a, jukefix@1.0.0, yungsapi@1.16.4-Fabric-1, randompatches@2.1.5-fabric, cardinal-components-level@2.7.10, org_jetbrains_kotlin_kotlin-stdlib@1.4.21, fabric-blockrenderlayer-v1@1.1.4+6a2618f53a, org_jetbrains_kotlin_kotlin-stdlib-jdk7@1.4.21, fabric@0.29.3+1.16, cardinal-components-scoreboard@2.7.10, fabric-networking-v0@0.3.1+2a4333d33a, konkrete@1.1.0, fabric-screen-handler-api-v1@1.1.0+6a2618f53a, modmenu@1.14.13+build.19, phosphor@0.7.0+build.10, cloth-utils-v1@1.4.9, fabric-tool-attribute-api-v1@1.2.5+6a2618f53a, org_jetbrains_kotlin_kotlin-stdlib-jdk8@1.4.21, roughlyenoughitems-runtime@5.8.9, cloth-client-events-v0@1.4.9, fabric-events-interaction-v0@0.4.1+6a2618f53a, fabric-crash-report-info-v1@0.1.2+b7f9825d3a, spark@1.4.3, fabric-language-kotlin@1.4.21+build.1, nbtcrafting@2.0.3+mc1.16.4, mousewheelie@1.6.1+mc1.16.4, fabric-key-binding-api-v1@1.0.1+730711c63a, cloth-config2@4.8.3, disable_custom_worlds_advice@1.2, org_jetbrains_kotlinx_kotlinx-coroutines-jdk8@1.4.2, amecs@1.3.2+mc.1.16.3, fabric-resource-loader-v0@0.4.1+0d4d7e343a, chunkpregen@0.3.3, org_jetbrains_kotlin_kotlin-reflect@1.4.21, antighost@1.16.4-fabric0.25.1-1.1.3, cardinal-components-entity@2.7.10, fabric-commands-v0@0.2.1+cbe9176f3a, fabric-registry-sync-v0@0.7.3+be155ae23a, cardinal-components-base@2.7.10, fabric-mining-levels-v0@0.1.2+6a2618f53a, fabric-networking-api-v1@1.0.0+4358fbc63a, fabric-item-groups-v0@0.2.2+cbe9176f3a, fabric-models-v0@0.2.0+cbe9176f3a, org_jetbrains_annotations@20.0.0, lithium-api@0.6.0, fabric-renderer-api-v1@0.4.0+cbe9176f3a, cardinal-components-block@2.7.10, fabric-structure-api-v1@1.1.3+cbe9176f3a, lithium@0.6.0, fabric-containers-v0@0.1.9+a03e98793a, fabric-object-builder-api-v1@1.9.2+6a2618f53a, fabric-rendering-data-attachment-v1@0.1.4+6a2618f53a, cloth-dynamic-registry-api-v1@1.4.9, cardinal-components-item@2.7.10, fabric-textures-v0@1.0.5+6a2618f53a, com_electronwill_night-config_core@3.6.3, fabric-rendering-fluids-v1@0.1.12+6a2618f53a, cloth-scissors-api-v1@1.4.9, fabric-lifecycle-events-v1@1.2.0+ffb68a873a, cardinal-components-chunk@2.7.10, sodium@0.1.0, cardinal-components-util@2.7.10, fabric-renderer-registries-v1@2.2.0+6a2618f53a, fabric-loot-tables-v1@1.0.1+6a2618f53a, cloth-armor-api-v1@1.4.9, dynamicfps@1.2.1, blame@2.2.6, fabric-particles-v1@0.2.3+cbe9176f3a, autoconfigtoml@1.0.0-fabric, minecraft@1.16.4, roughlyenoughitems-default-plugin@5.8.9, fabric-networking-blockentity-v0@0.2.7+a03e98793a, cloth-basic-math@0.5.1, fabric-command-api-v1@1.0.9+6a2618f53a, fatxporbs@0.0.9, amecsapi@1.1.1+mc1.16-rc1, inventoryhotswap@1.0, advdebug@2.2.0, fabric-renderer-indigo@0.4.3+6a2618f53a, org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm@1.4.2, smoothboot@1.16.4-1.4.0, com_electronwill_night-config_toml@3.6.3, dark-loading-screen@1.5.0, fabric-item-api-v1@1.2.0+6a2618f53a, roughlyenoughitems@5.8.9, cardinal-components@2.7.10, fabric-entity-events-v1@1.0.2+8004b36d3a, autoreconnect@1.1.0, fabric-rendering-v0@1.1.1+6a2618f53a, fabric-rendering-v1@1.5.0+c26373133a, fastbench@2.3, autoconfig1u@3.3.1, fabric-content-registries-v0@0.2.0+e77439c73a, cardinal-components-world@2.7.10, fabric-tag-extensions-v0@1.1.0+e77439c73a, fastfurnace@2.3, fabric-biome-api-v1@3.1.0+2e23b97c3a, itemmodelfix@0.2.4, cloth-api@1.4.9, cloth-datagen-api-v1@1.4.9, fabric-events-lifecycle-v0@0.2.0+16acbe5ba7, fabric-object-builders-v0@0.7.1+6a2618f53a, tweed@3.0.0-beta.22
[22:53:10] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/E:/minecraft/multiMC/mmc-stable-win32/MultiMC/libraries/net/fabricmc/sponge-mixin/0.8.2+build.24/sponge-mixin-0.8.2+build.24.jar Service=Knot/Fabric Env=CLIENT
[22:53:10] [main/INFO]: Loaded configuration file for Sodium (38 options available, 0 user overrides)
[22:53:10] [main/INFO]: Sodium has been successfully discovered and initialized -- your game is now faster!
[22:53:10] [main/INFO]: Loaded configuration file for Lithium: 63 options available, 0 override(s) found
[22:53:10] [main/INFO]: FabricZero: Loaded!
[22:53:11] [main/WARN]: @Mixin target me/shedaniel/rei/server/RecipeFinder is public in nbtcrafting.compat.mixins.json:MixinREIRecipeFinder and should be specified in value
[22:53:11] [main/WARN]: @Mixin target net/minecraft/class_3898$class_3208 is public in fabric-networking-api-v1.mixins.json:accessor.EntityTrackerAccessor and should be specified in value
[22:53:11] [main/INFO]: Smooth Boot config initialized
[22:53:16] [main/ERROR]: Write access detected to @Final field field_1705:Lnet/minecraft/class_329; in inventoryhotswap.mixins.json:HideHudMixin::handler$zme000$onHudRender
[22:53:16] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[22:53:17] [main/INFO]: Setting user: Josheva
[22:53:18] [main/INFO]: [STDOUT]: [KONKRETE] Successfully initialized!
[22:53:18] [main/ERROR]: Blame 2.2.6 initialized
[22:53:18] [main/ERROR]: [AutoReconnect] NoSuchFileException: E:\minecraft\multiMC\mmc-stable-win32\MultiMC\instances\Single Mod Test\.minecraft\config\autoreconnect.json
[22:53:18] [main/INFO]: [AutoReconnect] Creating default config...
[22:53:18] [main/INFO]: [AutoReconnect] Saved config
[22:53:18] [Async Class PreLoader/INFO]: Preloading classes...
[22:53:18] [main/INFO]: [STDOUT]: E:\minecraft\multiMC\mmc-stable-win32\MultiMC\libraries\com\mojang\minecraft\1.16.4\minecraft-1.16.4-client.jar
[22:53:19] [main/INFO]: Mod 'cotton-client-commands' uses the modmenu:clientsideOnly custom value unnecessarily, as it can be inferred from the mod's declared environment.
[22:53:19] [main/INFO]: Mod 'cloth-scissors-api-v1' uses the modmenu:clientsideOnly custom value unnecessarily, as it can be inferred from the mod's declared environment.
[22:53:19] [main/INFO]: Config loaded.
[22:53:19] [main/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[22:53:19] [main/INFO]: Reloaded configs for mousewheelie (universal/highest)
[22:53:19] [main/INFO]: Backend library: LWJGL version 3.2.2 build 10
[22:53:21] [main/INFO]: Narrator library for x64 successfully loaded
[22:53:21] [main/INFO]: Reloading ResourceManager: Default, Fabric Mods (Fabric Key Bindings (v0), Fabric Loader, Fabric Zero, Fabric Game Rule API (v1), Fabric API Base, RandomPatches, Fabric BlockRenderLayer Registration (v1), Fabric API, Fabric Networking (v0), Konkrete, Fabric Screen Handler API (v1), Mod Menu, Phosphor, Fabric Tool Attribute API (v1), REI (Runtime), Fabric Events Interaction (v0), Fabric Crash Report Info (v1), spark, Fabric Language Kotlin, NBT Crafting, Mouse Wheelie, Fabric Key Binding API (v1), Cloth Config v4, Disable Custom Worlds Advice, Amecs, Fabric Resource Loader (v0), AntiGhost, Fabric Commands (v0), Fabric Registry Sync (v0), Fabric Mining Levels (v0), Fabric Networking API (v1), Fabric Item Groups (v0), Fabric Models (v0), Fabric Renderer API (v1), Fabric Structure API (v1), Lithium, Fabric Containers (v0), Fabric Object Builder API (v1), Fabric Rendering Data Attachment (v1), Fabric Textures (v0), Fabric Rendering Fluids (v1), Fabric Lifecycle Events (v1), Sodium, Fabric Renderer Registries (v1), Fabric Loot Tables (v1), Dynamic FPS, Blame, Fabric Particles (v1), AutoConfig-TOML, Fabric Networking Block Entity (v0), Fabric Command API (v1), Amecs API, Inventory Hotswap, Advancements Debug, Fabric Renderer - Indigo, Smooth Boot, Dark Loading Screen, Fabric Item API (v1), Roughly Enough Items, Fabric Entity Events (v1), Fabric Rendering (v0), Fabric Rendering (v1), Auto Config v1 Updated, Fabric Content Registries (v0), Fabric Tag Extensions (v0), FastFurnace, Fabric Biome API (v1), Item Model Fix, Fabric Events Lifecycle (v0), Fabric Object Builders (v0), Tweed API)
[22:53:21] [main/INFO]: [STDOUT]: Inject generated resource packs.
[22:53:21] [main/INFO]: [STDOUT]: [KONKRETE] Ready to use!
[22:53:22] [Async Class PreLoader/INFO]: Preloaded 5445 classes!
[22:53:27] [main/INFO]: OpenAL initialized.
[22:53:27] [main/INFO]: Sound engine started
[22:53:27] [main/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/blocks.png-atlas
[22:53:27] [main/INFO]: Created: 256x128x4 minecraft:textures/atlas/signs.png-atlas
[22:53:27] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[22:53:27] [main/INFO]: Created: 1024x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[22:53:27] [main/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[22:53:27] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[22:53:27] [main/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[22:53:28] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/particles.png-atlas
[22:53:28] [main/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[22:53:28] [main/INFO]: Created: 256x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[22:53:28] [main/INFO]: Reloaded configs for mousewheelie (client/smallest)
```

**Comment by Bstn1802** [comment]
Yes, its logging the error just for information purposes and then creates the default config. The average user wont see that anyway, i just kept it that way since it might help to understand whats happening in case of a "real" issue. I agree, i could change it to warning and i might do that.
